en/config_vtol/vtol_back_transition_tuning.md

```diff
@@ -1,35 +1,55 @@
 
 ## Setting expected deceleration
+
 When flying missions that make use of a [VTOL_LAND](http://mavlink.org/messages/common#MAV_CMD_NAV_VTOL_LAND) waypoint the autopilot will attempt to calculate the proper distance at which to initiate the back-transition. It does this by looking at the current velocity (comparable to ground speed) and the expected deceleration. To get the vehicle to come out of back-transition very close to its landing point you can tune the expected deceleration ([VT_B_DEC_MSS](../advanced_config/parameter_reference.md#VT_B_DEC_MSS)) parameter. Make sure you have a large enough back-transition duration to allow the vehicle to reach its intended position before this timeout kicks in.
 
 ## Applying airbrakes
-If your vehicle is equipped with airbrakes and your selected airframe (mixer) is configured to make use of these you can set the airbrake position during back-transition in [VT_B_REV_OUT](../advanced_config/parameter_reference.md#VT_B_REV_OUT). The values scale from 0 to 1, so a value of 0.7 equals 70% output.
+
+If your vehicle is equipped with airbrakes, and your selected airframe supports them (in code), you can set the airbrake position during back-transition in [VT_B_REV_OUT](../advanced_config/parameter_reference.md#VT_B_REV_OUT). The values scale from 0 to 1, so a value of 0.7 equals 70% output.
 
 ## Applying reverse thrust on your fixed wing motor
-To get the shortest possible back-transition PX4 supports active breaking by reversing the fixed wing motor direction. To use this feature you will require an ESC that supports motor rotation reversing. Generally there are 2 ways a reverse capable ESC can implement this. 
+
+To get the shortest possible back-transition, PX4 supports active breaking by reversing the fixed wing motor direction. To use this feature you will require an ESC that supports motor rotation reversing. 
+
 > **Note** A typical fixed wing propeller is not optimized to spin in reverse, when the throttle during reverse thrust is set too high the propeller can stall.
 
+Generally there are 2 ways a reverse-capable ESC can implement reverse thrust. 
+
+
 ### Using throttle scaling (3D)
-3D ESCs assume 0 thrust at 50% throttle, positive (forward) thrust above 50% and negative thrust (reverse) below 50%. The airframe (mixer) can be configured to implement this so that the throttle stick can be used from 0% to 100% purely for forward thrust and only during back-transition will implement negative (reverse) thrust. To achieve this you must set the [VT_B_TRANS_THR](../advanced_config/parameter_reference.md#VT_B_TRANS_THR) parameter to a negative value between 0 and -1.
+
```

Comment:
@sanderux  Most of this is pretty much minor typos, but this section had a bit more of a restructure. I'm going to merge to make sure this gets published, but would be good if you could just scan this PR.